src/containers/bitset_containers.c

```diff
@@ -0,0 +1 @@
+#include <../src/containers/bitset.c>
```

Comment:
Same here, a comment would help on why this file exists. I think it would be better to use "bitset.c" instead of this relative path from the include files as this is a little ambiguous.

Reply:
If I only put `#include <bitset.c>` I get a "Cannot open include file: 'bitset.c': No such file" error. It seem to require this path, especially the `..` as just `src/containers/bitset.c` does not work either.

Reply:
Using `"bitset.c"` instructs the preprocessor to check the current directory first where using `<bitset.c>` forces the preprocessor to avoid looking in the current directory.

Reply:
Ah, thanks. Yes, that fixed it.